marklogic-data-hub-central/ui/src/components/explore/entity-specific-sidebar/entity-specific-sidebar.tsx

```diff
@@ -0,0 +1,364 @@
+import React, {useContext, useEffect, useState} from "react";
+import styles from "./entity-specific-sidebar.module.scss";
+import Facet from "@components/facet/facet";
+import DateFacet from "@components/date-facet/date-facet";
+import DateTimeFacet from "@components/date-time-facet/date-time-facet";
+// import {HCInput} from "@components/common";
+// import {faSearch} from "@fortawesome/free-solid-svg-icons";
+import moment from "moment";
+import {SearchContext} from "../../../util/search-context";
+import DynamicIcons from "@components/common/dynamic-icons/dynamic-icons";
+
+interface Props {
+  entitySelected: any;
+  entityFacets: any,
+  checkFacetRender: (facets: any) => void;
+  facetRender: (facets: any) => void;
+  updateSpecificFacets: any;
```

Comment:
Can we start doing as Luciana mentioned and add types if possible? 
For example, updateSpecificFacets should always be a boolean, right?

Reply:
@aniamy I added type for updateSpecificFacets it is always a boolean.